Makefile

```diff
@@ -388,7 +472,7 @@ create-management-cluster: $(KUSTOMIZE) $(ENVSUBST)
 .PHONY: create-workload-cluster
 create-workload-cluster: $(KUSTOMIZE) $(ENVSUBST)
 	# Create workload Cluster.
-	$(KUSTOMIZE) build templates | $(ENVSUBST) | kubectl apply -f -
+	cat templates/cluster-template.yaml | $(ENVSUBST) | kubectl apply -f -
```

Comment:
Could we potentially use clusterctl config in the future?

Reply:
I'm not sure how these are being used by people, so didn't want to change too much. It was being used by the conformance test (where it would always build in the CI download script), which will become irrelevant.